### DISCUSSION TITLE
[QUESTION]  Send a simple message on a channel

### DISCUSSION THREAD

**Discussion by alexandrebouttier** [python-telegram-bot/python-telegram-bot#2278] [Q&A]
Hello, I'm a little lost in the doc, I would just like to send a message on a channel, could someone give me a complete example with the right imports etc, thanks

**Comment by Bibo-Joshi** [comment]
```python
from telegram import Bot

bot = Bot('TOKEN')
bot.send_message(channel_id, message)
```

**Comment by alexandrebouttier** [comment]
I come across this error    ImportError: cannot import name 'Bot' from partially initialized module 'telegram' (most likely due to a circular import

```  
from telegram import Bot

channel_id="@mychanel"
bot = Bot('mytoken')
bot.send_message(channel_id, "test") 
```

**Comment by Bibo-Joshi** [comment]
then `telegram` is not the module provided by `python-telegram-bot`. Make sure not to name any files or directories of your bot `telegram`.

**Comment by alexandrebouttier** [comment]
I no longer have the tax error, create a virtual environment, but now I come across this error while I am using the right channel 

``` 
telegram.error.BadRequest: Chat not found
```

**Reply by Bibo-Joshi** [reply]
That means that you're either passing the wrong `chat_id` or that the bot does not have write permissions for that chat.

**Reply by alexandrebouttier** [reply]
i followed this to get the channel number https://gist.github.com/mraaroncruz/e76d19f7d61d59419002db54030ebe35

**Reply by Bibo-Joshi** [reply]
is the bot admin of that channel? if not, it can't send messages to it

**Reply by alexandrebouttier** [reply]
thanks
